src/targets/gpu/CMakeLists.txt

```diff
@@ -236,6 +240,13 @@ else()
     message(STATUS "MIOpen does not have find mode api")
 endif()
 
+if(HAS_ROCBLAS_BETA_FEATURES_API)
+    target_compile_definitions(migraphx_gpu PUBLIC -DROCBLAS_BETA_FEATURES_API -DROCBLAS_NO_DEPRECATED_WARNINGS)
+    message(STATUS "MIGraphx is using Beta API of MIOpen")
```

Comment:
This is the beta API of rocblas not miopen.

Reply:
changed